application/console/views/cron/scripts/upload/default/publish.sh

```diff
@@ -288,6 +289,76 @@ prepare_publish() {
   fi
 }
 
+notify_update_json() {
+  local i_json="$1"
+  local i_type="$2"
+  local i_url="$3"
+
+  echo "${i_json}" | jq \
+    --arg type "${i_type}" \
+    --arg idx "${SCRIPTURE_EARTH_ID}" \
+    --arg url "${i_url}" \
+    --arg email "${PROJECT_OWNER_EMAIL}" \
+    --arg projectName "${PROJECT_NAME}" \
+    --arg projectDescription "${PROJECT_DESCRIPTION}" \
+    --arg username "${PROJECT_OWNER_NAME}" \
+    --arg organization "${PROJECT_ORGANIZATION}" \
+    --arg project "${PROJECT_URL}" \
+    '. + [ { type: $type, idx: $idx, url: $url, email: $email, projectName: $projectName, projectDescription: $projectDescription, username: $username, organization: $organization}]'
+}
+
+notify_scripture_earth() {
+  local notify_json="$1"
+  # curl does not work! SE.org returns 406 Not Acceptable and can't figure out why.
+  wget \
+     --header "Content-Type: application/json" \
+     --header "Accept: application/json" \
+     --post-data "${notify_json}" \
+     "https://scriptureearth.org/api/add_resource.php?v=1&key=${SCRIPTURE_EARTH_KEY}"
+
+}
+
+post_publish() {
```

Comment:
Add comment on WORKFLOW_PRODUCT_NAME being user generated name and iOS, android, google, and PWA having to be in the name for this to work.